grove/grovetccfg/grovetccfg.go

```diff
@@ -259,20 +328,162 @@ func main() {
 	os.Exit(0)
 }
 
-func createRulesOldAPI(toc *to.Session, host string, certDir string) (remap.RemapRules, error) {
-	cachegroupsArr, err := toc.CacheGroups()
+func createGroveCfg(toc *to.Session, server tc.Server) (bool, config.Config, error) {
+	var new_cfg config.Config
+	var cur_cfg config.Config
+	var plugin_params = []string{}
+
+	// load the servers current config parameters.
+	if _, err := os.Stat(GroveConfigPath); err == nil {
+		cur_cfg, err = config.LoadConfig(GroveConfigPath)
+		if err != nil {
+			fmt.Println(time.Now().Format(time.RFC3339Nano) + " Error loading current config from '" + GroveConfigPath + "' " + err.Error())
+			return false, cur_cfg, err
+		} else {
+			// make sure this array is sorted for later comparison
+			sort.Strings(cur_cfg.Plugins)
+		}
+	}
+
+	serverParameters, err := toc.Parameters(server.Profile)
 	if err != nil {
-		fmt.Println(time.Now().Format(time.RFC3339Nano) + " Error getting Traffic Ops Cachegroups: " + err.Error())
-		os.Exit(1)
+		fmt.Println(time.Now().Format(time.RFC3339Nano) + " Error getting Traffic Ops Parameters for host '" + server.HostName + "' profile '" + server.Profile + "': " + err.Error())
+		return false, cur_cfg, err
+	} else {
+		// load config parameters from the servers profile
+		for _, p := range serverParameters {
+			if p.ConfigFile == "grove.cfg" {
```

Comment:
The config file is duplicated in `GroveConfigPath`. Can you change the variables to 
```
const GroveConfigFile = "grove.cfg"
const GroveConfigPath = "/etc/grove/" + GroveConfigFile
```

To get rid of the magic string?